### ISSUE TITLE
ErickSkrauch/align_multiline_parameters not working correctly with nullable type hints

### ISSUE THREAD

**Issue by manuelkiessling** [erickskrauch/php-cs-fixer-custom-fixers#3]
Hi @erickskrauch,

thanks for a great project, exactly what I'm looking for.

I think I might have encountered an issue, though.

I'd like to use `ErickSkrauch/align_multiline_parameters`, but it seems to be irritated when type hints are nullable, as in `?string` or `?RequestStack`.

This is the change it proposed, for a function declaration that already adheres to the align_multiline_parameters rules:

```
     public function __construct(
         EntityManagerInterface         $entityManager,
         LoggerInterface                $logger,
-        ?RequestStack                  $requestStack,
+        ?RequestStack                              $requestStack,
         WebDeveloperToolsHelperService $webDeveloperToolsHelperService
```

**Comment by erickskrauch** [comment]
Thank you for the report. I have fixed this issue and released it with the 1.2.1 tag.

**Comment by manuelkiessling** [comment]
Thank you so much @erickskrauch, works perfectly!
